### DISCUSSION TITLE
FEATURE REQUEST:Import RSS links from file like (opml ,json ,xml .etc)

### DISCUSSION THREAD

**Discussion by MohamedAhmyda** [dailydotdev/daily#535] [Feature Requests]
Adding the feature to import links from files like (opml ,json ,xml .etc) The important thing is that we can group links from one file. This will be a great feature, and I think everyone will agree with me in my opinion.
And I have this file that about 1000 RSS links to think about add it your feeds and if author one want to import it of course if the feature have added in future.

[feeds.zip](https://github.com/dailydotdev/daily/files/8643719/feeds.zip)

**Comment by idoshamun** [comment]
Hi @MohamedAhmyda,

Thank you for your suggestion. At this point, we support submitting a single source at a time.
You can check out our guide: https://docs.daily.dev/docs/for-content-creators/suggest-new-source

Please know that every submission goes through a manual review.
